plugins/src/main/java/org/smartregister/fhir/gateway/plugins/PractitionerDetailsEndpointHelper.java

```diff
@@ -546,7 +546,7 @@ private List<OrganizationAffiliation> mapBundleToOrganizationAffiliation(
                 : Collections.emptyList();
     }
 
-    private List<LocationHierarchy> getLocationsHierarchy(List<String> locationsIdentifiers) {
+    static List<LocationHierarchy> getLocationsHierarchy(List<String> locationsIdentifiers) {
```

Comment:
You can expand the accessibility by updating the access modifier to `public`
```suggestion
    public static List<LocationHierarchy> getLocationsHierarchy(List<String> locationsIdentifiers) {
```